igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildCache.groovy

```diff
@@ -38,16 +38,18 @@ class JenkinsCache {
     @Value('${spinnaker.jedis.prefix:igor}')
     String prefix
 
+    String id = 'builds'
+
     List<String> getJobNames(String master) {
         Jedis resource = jedisPool.resource
-        List<String> jobs = resource.keys("${prefix}:${master}:*").collect { extractJobName(it) }.sort()
+        List<String> jobs = resource.keys("${prefix}:${id}:${master}:*").collect { extractJobName(it) }.sort()
```

Comment:
I will reinsert the JenkinsCache and make BuildCache have a "ci system" field in the key. So that we can use the same cache if another ci system gets added.